src/sna/toots-sna.ipynb

```diff
@@ -268,16 +214,234 @@
     "Median Replies: {}\"\"\".format(total_toots, with_replies_pct, median))\n",
     "\n",
     "median_degree = degree_sequence[int(len(degree_sequence) / 2)]\n",
+    "avg_degree = sum(degree_sequence)/len(degree_sequence)\n",
+    "\n",
     "total_threads = (statuses_df['parent_account_id'] != statuses_df['parent_account_id'].shift(axis=0)).sum(axis=0)\n",
     "# closeness = nx.closeness_centrality(G) # This is slow\n",
     "# Average Closness {} hops - need to convert to hops - otherwise doesn't make sense\n",
-    "cluster_coefficient = nx.average_clustering(G)\n",
+    "cluster_coefficient = nx.average_clustering(status_gph)\n",
     "\n",
     "\n",
     "print(\"\"\"\n",
     "Total Threads: {} \n",
-    "Median Degree: {} conn/p\n",
-    "Cluster Coefficient {}%\"\"\".format(total_threads, median_degree, round(cluster_coefficient * 100, 2)))\n"
+    "Median Degree: {} connn/pp\n",
+    "Avg Degree: {}\n",
+    "Cluster Coefficient {}%\"\"\".format(total_threads, median_degree,  avg_degree, round(cluster_coefficient * 100, 2)))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c1e012a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def draw_kamada(G):\n",
+    "    nodes = G.nodes()\n",
+    "    degree = G.degree()\n",
+    "    colors = [degree[n] for n in nodes]\n",
+    "    pos = nx.kamada_kawai_layout(G)\n",
+    "    #pos = nx.spring_layout(G, k = 0.2)\n",
+    "    cmap = plt.cm.viridis_r\n",
+    "    cmap = plt.cm.Greys\n",
+    "\n",
+    "    vmin = min(colors)\n",
+    "    vmax = max(colors)\n",
+    "\n",
+    "    fig = plt.figure(figsize = (15,9), dpi=100)\n",
+    "\n",
+    "    nx.draw(G,pos,alpha = 0.8, nodelist = nodes, node_color = 'w', node_size = 10, with_labels= False,font_size = 6, width = 0.2, cmap = cmap, edge_color ='yellow')\n",
```

Comment:
```suggestion
    "    nx.draw_networkx(G,pos,alpha = 0.8, nodelist = nodes, node_color = 'b', node_size = 10, with_labels= False,font_size = 6, width = 0.2, cmap = cmap, edge_color ='green')\n",
```

Reply:
was getting this [error](https://stackoverflow.com/questions/74189581/axesstack-object-is-not-callable-while-using-networkx-to-plot)
Also changing the colors of nodes and edges here to make it visual friendly